runtime/testing-basilisk/src/lib.rs

```diff
@@ -893,10 +893,23 @@ impl pallet_route_executor::Config for Runtime {
 	type Balance = Balance;
 	type MaxNumberOfTrades = MaxNumberOfTrades;
 	type Currency = MultiInspectAdapter<AccountId, AssetId, Balance, Balances, Tokens, NativeAssetId>;
-	type AMM = (XYK, LBP);
+	type AMM = (XYK, LBP, Stableswap);
 	type WeightInfo = common_runtime::weights::route_executor::BasiliskWeight<Runtime>;
 }
 
+impl pallet_stableswap::Config for Runtime {
+	type Event = Event;
+	type AssetId = AssetId;
+	type Currency = Currencies;
+	type ShareAccountId = AssetsAccountId<Self>;
+	type AssetRegistry = AssetRegistry;
+	type AuthorityOrigin = SuperMajorityTechCommitteeOrRoot;
+	type MinPoolLiquidity = StableswapMinPoolLiquidity;
+	type MinTradingLimit = StableswapMinTradingLimit;
+	type AmplificationRange = AmplificationRange;
+	type WeightInfo = (); //TODO:
```

Comment:
todo

Reply:
yes, this is only draft. There are missing benchmarks in the warehouse. I'll implement it and fix these but I'm working on something else for now.